<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>2</y>
      <width>526</width>
      <height>473</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="repositoryListLabel">
           <property name="text">
            <string>Repository dogs</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="repositoryListWidget"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="nameLabel">
                 <property name="text">
                  <string>Name:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="nameInput"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="breedLabel">
                 <property name="text">
                  <string>Breed:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="breedInput"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="urlLabel">
                 <property name="text">
                  <string>URL:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="urlInput"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="ageLabel">
                 <property name="text">
                  <string>Age:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="ageInput"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="idLabel">
               <property name="text">
                <string>ID:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="idInput"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="addButton">
               <property name="text">
                <string>Add</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="adoptionListLabel">
           <property name="text">
            <string>Adoption List</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="adoptionListWidget"/>
         </item>
         <item>
          <widget class="QLabel" name="filterListLabel">
           <property name="text">
            <string>Filter List</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="filterListWidget"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="breedFilterLabel">
               <property name="text">
                <string>Breed Filter:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="breedFilterInput"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QPushButton" name="filterButton">
               <property name="text">
                <string>Filter</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="ageFilterLabel">
               <property name="text">
                <string>Age Filter</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="ageFilterInput"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QPushButton" name="adoptButton">
           <property name="text">
            <string>Adopt</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="adoptionIdInput">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QPushButton" name="deleteButton">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="removeIdInput"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
